src/main/kotlin/no/nav/syfo/syfosmvarsel/Bootstrap.kt

```diff
@@ -56,24 +63,36 @@ fun main() {
     val applicationServer = ApplicationServer(applicationEngine, applicationState)
     applicationServer.start()
 
-    val kafkaBaseConfig = loadBaseConfig(env, vaultSecrets)
-            .envOverrides()
-    val consumerProperties = kafkaBaseConfig.toConsumerConfig(
-            "syfosmvarsel-consumer", valueDeserializer = StringDeserializer::class
-    )
-
-    val producerProperties = kafkaBaseConfig.toProducerConfig(
-            "syfosmvarsel", valueSerializer = JacksonKafkaSerializer::class)
+    val kafkaBaseConfig = loadBaseConfig(env, vaultSecrets).envOverrides()
 
     val diskresjonskodeService = createPort<DiskresjonskodePortType>(env.diskresjonskodeEndpointUrl) {
         port { withSTS(vaultSecrets.serviceuserUsername, vaultSecrets.serviceuserPassword, env.securityTokenServiceURL) }
     }
 
-    val kafkaProducer = KafkaProducer<String, OppgaveVarsel>(producerProperties)
-    val varselProducer = VarselProducer(diskresjonskodeService, kafkaProducer, env.oppgavevarselTopic)
+    val varselProducer = getVarselProducer(kafkaBaseConfig, env, diskresjonskodeService)
+    val avvistKafkaConsumer = getAvvistKafkaConsumer(kafkaBaseConfig, env)
```

Comment:
Vi kan kanskje bruke 1 consumer for avviste og nye?

Reply:
Hvis det er samme consumer som leser fra alle tre topicene, hvordan vet jeg forskjellen på om jeg har lest en sykmelding som er ok/manuell og en som er avvist..? De skal jo håndteres helt ulikt :)